### PR TITLE
v1.1 player

### PR DESCRIPTION
**Pull request by jgolden**

### CODE REVIEW

**Comment by dudaski** [comment]
:100:

**Comment by serv** [comment]
@dudaski You can add reactions now

**Comment by dudaski** [comment]
@serv they need more reactions tho

File: README.md [hunk 1]
```diff
@@ -30,51 +30,38 @@ Rhapsody.member.set({
...
-#### Playing a track
-```javascript
-Rhapsody.player.play('Tra.5156528');
-```
-
-#### Pausing
-```javascript
-Rhapsody.player.pause();
-```
+### Check if player is ready and authorize
```
Comment (serv) [inline README.md]: Just one little change from `authorize` to `authorized`.

File: rhapsody.js [hunk 2]
```diff
@@ -268,7 +268,7 @@
       seek: function(t) {
         Rhapsody.windows(this.win).post('seek', t);
       },
-
+
```
Comment (serv) [inline rhapsody.js]: And we can get rid of this whitespace.

**Comment by aguywithanidea** [comment]
@jgolden - have you been able to sync with @serv on this?

**Pull request merged**